migrations/20250912000000_fix_uuid_nil_postgres15.up.sql

```diff
@@ -0,0 +1,10 @@
+-- Fix for PostgreSQL 15 compatibility: Define uuid_nil() function if it doesn't exist
+-- In PostgreSQL 15, uuid_nil() may not be available even with uuid-ossp extension
+
+-- Patch: Ensure uuid_nil() exists for Postgres 15+ without uuid-ossp
+
+-- Patch: Always (re)define uuid_nil() for Postgres 14/15 compatibility
```

Comment:
todo: remove duplicate comment. had an issue with rebasing from different remotes, this snuck past